### PR TITLE
feat(README) : Add info over how to do the releases (AEROGEAR-9372)

### PR DESCRIPTION
**Pull request by camilamacedo86**
## Motivation
https://issues.jboss.org/browse/AEROGEAR-9372

## What
- Fix tag name in the CHANGELOG
- Add info to the README over how to do the releases

## Why
- Have this process doc

## Checklist:

- [X] Code has been tested locally by PR requester
- [ ] Changes have been successfully verified by another team member 

## Progress

- [X] Finished task
- [ ] TODO

### CODE REVIEW

File: CHANGELOG.MD [hunk 1]
```diff
@@ -8,5 +8,5 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## Released
 
-## [v0.1.0] - Tue Jun 11 2019
```
Comment (damienomurchu) [inline CHANGELOG.MD]: Should the initial version be `0.1.0` or `1.0.0`?

File: README.adoc [hunk 2]
```diff
@@ -409,6 +409,30 @@ NOTE: The link:./Makefile[Makefile] is implemented with tasks which you should u
...
+Following the steps
+
+* Create a new tag following the http://semver.org/spec/v2.0.0.html[semver]. Following an example.
+
+[source,shell]
+----
+$ git tag -a 0.1.0 -m "version 0.1.0"
+----
+
+* Push the new tag to the upstream repository.  Following an example.
```
Comment (damienomurchu) [inline README.adoc]:
```suggestion
* Push the new tag to the upstream repository, for example:
```

File: README.adoc [hunk 3]
```diff
@@ -409,6 +409,30 @@ NOTE: The link:./Makefile[Makefile] is implemented with tasks which you should u
 * https://golang.org/[Golang] - Programming language used
 * https://echo.labstack.com/[Echo] - Web framework used
 
+== Release
+
+Following the steps
+
+* Create a new tag following the http://semver.org/spec/v2.0.0.html[semver]. Following an example.
```
Comment (damienomurchu) [inline README.adoc]:
```suggestion
* Create a new tag following the http://semver.org/spec/v2.0.0.html[semver], for example:
```

File: README.adoc [hunk 4]
```diff
@@ -409,6 +409,30 @@ NOTE: The link:./Makefile[Makefile] is implemented with tasks which you should u
...
+[source,shell]
+----
+$ git push upstream 0.1.0
+----
+
+* Update the the link:./CHANGELOG.MD[CHANGELOG.MD] with the new release.
+
+NOTE: The image with the tag will be created and pushed to https://quay.io/repository/aerogear/mobile-security-service[quay.io/repository/aerogear/mobile-security-service] by the CI.
+
+WARNING: Do not use letters in the tag as `v`. It will not work.
```
Comment (damienomurchu) [inline README.adoc]:
```suggestion
WARNING: Do not use letters in the tag such as `v`. It will not work.
```

File: README.adoc [hunk 5]
```diff
@@ -409,6 +409,30 @@ NOTE: The link:./Makefile[Makefile] is implemented with tasks which you should u
...
+* Push the new tag to the upstream repository.  Following an example.
+
+[source,shell]
+----
+$ git push upstream 0.1.0
+----
+
+* Update the the link:./CHANGELOG.MD[CHANGELOG.MD] with the new release.
+
+NOTE: The image with the tag will be created and pushed to https://quay.io/repository/aerogear/mobile-security-service[quay.io/repository/aerogear/mobile-security-service] by the CI.
```
Comment (damienomurchu) [inline README.adoc]:
```suggestion
NOTE: The image with the tag will be created and pushed to the https://quay.io/repository/aerogear/mobile-security-service[aerogear image repository] by the CI.
```

File: README.adoc (hunk 4 above)
Comment (camilamacedo86) [reply README.adoc]: Done.

File: README.adoc (hunk 3 above)
Comment (camilamacedo86) [reply README.adoc]: Done.

File: README.adoc (hunk 2 above)
Comment (camilamacedo86) [reply README.adoc]: Done.

File: CHANGELOG.MD (hunk 1 above)
Comment (camilamacedo86) [reply CHANGELOG.MD]: we spoke over it considering soved.,

File: README.adoc (hunk 5 above)
Comment (camilamacedo86) [reply README.adoc]: ddone

**Comment by laurafitzgerald** [comment]
Seems to be some duplication with https://github.com/aerogear/mobile-security-service-operator/pull/118 Wanna rebase before we do the review?

**Comment by camilamacedo86** [comment]
Hi @laurafitzgerald both have the CI automated. However, each project is a project and need to have your steps doc. Also, note that for the operator we have other steps and not these ones too.

**Comment by laurafitzgerald** [comment]
@camilamacedo86 yep, didn't spot that it was the service repo and not the operator one. What are the verification steps here?

**Comment by camilamacedo86** [comment]
@laurafitzgerald verification steps to doc the process in the readme?

**Comment by craicoverflow** [comment]
Regarding a changelog - a benefit of conventional commits is the ability to generate change logs from commits.

Is this something we have tried?

https://github.com/conventional-changelog/conventional-changelog

**Comment by davidffrench** [comment]
LGTM @camilamacedo86 

@craicoverflow I am happy with the manual changelog for now and have our good commit messages be part of a release in GitHub.

**Comment by damienomurchu** [comment]
> Regarding a changelog - a benefit of conventional commits is the ability to generate change logs from commits.
> 
> Is this something we have tried?
> 
> https://github.com/conventional-changelog/conventional-changelog

This is something we've discussed in the past and part of the reason we switched to using the commit message convention we use now. I feel the discussion may spiral beyond the scope of this PR, so have posted more detailed thoughts in our chat channel

**Pull request merged**